README.md

```diff
@@ -1,5 +1,110 @@
 # [BuckleScript](https://github.com/bloomberg/bucklescript) bindings for [React Native](https://github.com/facebook/react-native)
 
+## Getting started
+
+Great that you want to use Reason with React Native! To get everything running are just a couple of steps.
+Let's assume that you already have a React Native project. Otherwise follow the React Native [instructions](http://facebook.github.io/react-native/docs/getting-started.html) until you have your app running.
+
+1. Install [Bucklescript](https://github.com/bloomberg/bucklescript) (the Reason -> JS compiler), [Reason-React](https://github.com/reasonml/reason-react) and `bs-react-native`:
+```sh
+# substitute yarn with npm if you prefer
+yarn add bs-platform reason-react https://github.com/BuckleTypes/bs-react-native.git
+```
+
+2. Create a `re` folder (there will be your Reason code)
+3. Create a `bsconfig.json` with the following content file in your project root
+```json
+{
+    "name": "my-awesome-app",
+    "reason": {
+        "react-jsx": true
+    },
+    "bs-dependencies": ["bs-react-native", "reason-react"],
+    "sources": [{
+        "dir": "re"
+    }]
+}
+```
+4. You are nearly done, the last configuration before we get to the fun stuff. In your `package.json` add to the `"scripts"` section two scripts:
+
+```json
+"scripts": {
+  ...
+  "build": "bsb -make-world -clean-world",
+  "watch": "bsb -make-world -clean-world -w"
+}
+```
+Now you can build all your (so far nonexsisting) Reason code with `yarn run build` or enter the watch mode with `yarn run watch`.
+
+5. Start the watcher with `yarn run watch`.
+6. Now we come to the fun stuff! We will rebuild the default app component with Reason. Create a new file `re/app.re` and make it look like this:
+```reason
+open ReactNative;
```

Comment:
Maybe let's shorten this to a simple `Hello World`? I can see that you made it the same as `react native` intro but I think it's a lot of things a beginner might not know.

```reason
open ReactNative;
let helloWorld () => 
  <View style=Style.(style [flex 1., justifyContent `center, alignItems `center])>
    <Text value="Hello World" />
  </View>;
```

```js
 import { helloWorld } from "./lib/js/re/app.js";
 import React from "react";
 import {
   AppRegistry
 } from 'react-native';
 
 AppRegistry.registerComponent('MyAwesomeProject', helloWorld);
```